app/Models/Post.php

```diff
@@ -53,7 +53,22 @@ public function signup()
      */
     public function reactions()
     {
-        return $this->hasMany(Reaction::class);
+        // return $this->morphToMany(Reaction::class, 'postable');
+        return $this->morphMany(Reaction::class, 'postable');
+    }
```

Comment:
@DFurnes ah progress! When I change to just `return $this->morphTo(Reaction::class, 'postable');` it returns `null` when I die dump `$post->reactions` which makes sense however I added a reaction for a post and it still says `null`. 

With `return $this->morphMany(Reaction::class, 'postable');` and the `array_key_exists` error, the `$key` is `id`, which is a string so doesn't quite add up to the error message?